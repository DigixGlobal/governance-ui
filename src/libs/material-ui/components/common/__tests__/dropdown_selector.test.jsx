import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import DropdownSelector from '../dropdown_selector';

describe('<DropdownSelector />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <DropdownSelector
        items={[]}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
