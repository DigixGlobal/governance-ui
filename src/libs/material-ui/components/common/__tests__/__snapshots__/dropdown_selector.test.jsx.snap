// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownSelector /> renders correctly 1`] = `
<Dropdown
  additionLabel="Add "
  additionPosition="top"
  className="icon  "
  closeOnBlur={true}
  floating={true}
  fluid={true}
  icon="dropdown"
  labeled={true}
  minCharacters={1}
  noResultsMessage="No results found."
  openOnFocus={true}
  renderLabel={[Function]}
  searchInput="text"
  selectOnBlur={true}
  selectOnNavigation={true}
  selection={true}
  text="Select "
>
  <DropdownMenu>
    <DropdownMenu
      scrolling={true}
    />
  </DropdownMenu>
</Dropdown>
`;
